IDE/IAR-EWARM/Projects/user_settings.h

```diff
@@ -17,7 +17,11 @@
 
 #define WOLFSSL_GENSEED_FORTEST /* Warning: define your own seed gen */
 
-#define TFM_TIMING_RESISTANT
+/* Define math options */
```

Comment:
FYI: This was likely using integer.c heap math previously. Not fast math. I'd recommend providing a template here for users to choose 4 options. Use something like this: https://github.com/wolfSSL/wolfssl/blob/master/examples/configs/user_settings_template.h#L62